### DISCUSSION TITLE
Night mode

### DISCUSSION THREAD

**Discussion by Raynaudy** [ppoffice/hexo-theme-icarus#1131] [Q&A] [answered]
Hi everyone,

I saw online that there is a night theme on Icarus starting the 3.0 version.
I couldn't find the way to add the button to the navbar.
Could you help me? :) thanks!

**Comment by ppoffice** [comment, marked as answer]
@Raynaudy There is no official night mode currently. You may find useful tutorials on how to add that functionality online.

**Reply by Raynaudy** [reply]
Oh, thank you! I will stick with official version for now and might ask later if I couldn't find any english thread.
